asreviewcontrib/datatools/entrypoint.py

```diff
@@ -59,6 +60,43 @@ def execute(self, argv):
                     type=str,
                     help="Persistent identifier used for deduplication. Default: doi.",
                 )
+                dedup_parser.add_argument(
+                    "--drop_similar",
+                    action='store_true',
+                    help="Drop similar records.",
+                )
+                dedup_parser.add_argument(
+                    "--similarity",
+                    default=0.98,
+                    type=float,
+                    help="Similarity threshold for deduplication. Default: 0.98.",
+                )
+                dedup_parser.add_argument(
+                    "--skip_abstract",
+                    action='store_true',
+                    help="Use only title for deduplication.",
+                )
+                dedup_parser.add_argument(
```

Comment:
Maybe call this `skip_stopwords`, so that it's more similar to the command above?

Reply:
I was thinking of not removing the stopwords by default. Do you think it's best to leave it on by default?

Reply:
I was thinking of not removing the stopwords by default. Do you think it's best to leave it on by default?

Reply:
I agree, removing stopwords should not be on by default. My point was more that I thought the names are a bit confusing. There is `drop`, `skip` and `discard`, which all mean very similar things.

Reply:
What about:

- `--drop_similar` -> `--similar`
- `--skip_abstract` -> `--title_only`
- `--discard_stopwords` -> `--stopwords`
- `--strict_similarity` -> `--strict`

Reply:
Yes, I like it!

Reply:
Just applied these changes to the latest commit.